### DISCUSSION TITLE
Script Install - Which Steps to Follow??

### DISCUSSION THREAD

**Discussion by stevefxp1** [pfelk/pfelk#481] [Q&A]
Hello all,

I am trying to complete the scripted install of PFELK but I have a couple of questions:

1) Once all the pieces of PFELK are installed a dialog comes up that says I should skip and move to the security steps. Is this accurate? I should answer S?
2) How far into the security steps should I go? Do I stop after 6 is completed bc the file that step 7 says to edit is empty, when the example has more in it.

Thanks,
Steve

**Comment by gabriel25115CG** [comment]
> Hello all,
> 
> I am trying to complete the scripted install of PFELK but I have a couple of questions:
> 
>     1. Once all the pieces of PFELK are installed a dialog comes up that says I should skip and move to the security steps. Is this accurate? I should answer S?
> 
>     2. How far into the security steps should I go? Do I stop after 6 is completed bc the file that step 7 says to edit is empty, when the example has more in it.
> 
> 
> Thanks, Steve

Hey, first, you have to say "Yes" when they ask if you wan't skip and move the security steps. Secondly, the file at the 7 steps are not empty. You have to change the extention file like that : 50-outputs.conf ==> 50-outputs.pfelk 
Best regards
